zanata-war/src/main/java/org/zanata/security/ZanataIdentity.java

```diff
@@ -260,10 +616,66 @@ public String login(AuthenticationType authType) {
     public String getAccountUsername() {
         HAccount authenticatedAccount =
                 ServiceLocator.instance().getInstance(
-                        JpaIdentityStore.AUTHENTICATED_USER, HAccount.class);
+                        ZanataJpaIdentityStore.AUTHENTICATED_USER, HAccount.class);
         if (authenticatedAccount != null) {
             return authenticatedAccount.getUsername();
         }
         return null;
     }
+
+    public Principal getPrincipal() {
+        return principal;
+    }
+
+    public void checkRestriction(String expression) {
+        log.warn("permission check as EL is not supported yet");
+        throw new UnsupportedOperationException("identity.checkRestriction(exp) is not supported");
```

Comment:
- [ ] As far as I know, we don't plan to support EL for security checks at all, which means we shouldn't have any annotations which accept EL.  EL seems to be the purpose of `value` in `@CheckPermission`, but we hardly seem to be using `@CheckPermission` anywhere.  Is there some migration that we are missing?